src/ts/waterfall/sub-components/svg-marks.ts

```diff
@@ -1,8 +1,8 @@
-import {roundNumber} from "../../helpers/misc";
```

Comment:
It would be great to find a tslint rule for the spacing inside curly brackets (I haven't found one yet). I'm updating my editor config (IntelliJ) to put spaces inside imports and object literals. I'm thinking maybe we should split it into multiple PRs. One for the rounding fix, and one to apply consistent spacing within the entire code base.